runtime/integration-tests/src/liquidity_pools/pallet/development/tests/liquidity_pools/foreign_investments.rs

```diff
@@ -112,14 +112,6 @@ mod same_currencies {
 			let currency_id = AUSD_CURRENCY_ID;
 			let currency_decimals = currency_decimals::AUSD;
 
-			let sending_domain_locator = Domain::convert(DEFAULT_DOMAIN_ADDRESS_MOONBEAM.domain());
-			let pool_account =
-				pallet_pool_system::pool_types::PoolLocator { pool_id }.into_account_truncating();
-
-			Tokens::mint_into(AUSD_CURRENCY_ID, &sending_domain_locator, AUSD_ED).unwrap();
-			Tokens::mint_into(AUSD_CURRENCY_ID, &investor, AUSD_ED).unwrap();
-			Tokens::mint_into(AUSD_CURRENCY_ID, &pool_account, AUSD_ED).unwrap();
```

Comment:
Curious about why this is no longer needed

Reply:
When we increase a foreign investment, the corresponding amount is minted. 
https://github.com/centrifuge/centrifuge-chain/blob/90d3cf9f2884489dafbc221925b2608ced4ace5e/pallets/liquidity-pools/src/inbound.rs#L111

As long as the minted amount is above the ED, we are fine.

Reply:
This should be a requirement then for its usage, right? Nobody can send less than `ED`. Should this account be `Expendable` then?

Reply:
Yes and yes!

Reply:
Ok, then this should be a tech debt once we have everything passed and compiling